Combobox.js

```diff
@@ -722,19 +826,29 @@ define([
 					// Avoid that List gives focus to list items when navigating, which would
 					// blur the input field used for entering the filtering criteria.
 					this.dropDown.focusDescendants = false;
-
-					this._updateScroll(undefined, true);	// sets this.list.navigatedDescendant
-					this._setActiveDescendant(this.list.navigatedDescendant);
+					if (!this._useCenteredDropDown()) {
+						// desktop version
+						this._updateScroll(undefined, true);	// sets this.list.navigatedDescendant
+						this._setActiveDescendant(this.list.navigatedDescendant);
+					} else {
+						// mobile version
+						if (!this.hasDownArrow) {
+							if (filteringInProgress && this.inputNode.value.length >= this.minFilterChars) {
+								$(this.list).removeClass("d-combobox-list-hidden");
+							} else {
+								$(this.list).addClass("d-combobox-list-hidden");
+							}
```

Comment:
You could use the `d-hidden`/`d-shown` class or alternately set the `d-hidden`/`d-shown` attribute, rather than creating a new `d-combobox-list-hidden` class.   So this code could be changed to:

```js
this.list.setAttribute("d-shown", filteringInProgress &&
    this.inputNode.value.length >= this.minFilterChars);
```

Similarly of course for other code accessing `d-combobox-list-hidden`.